va/va_dec_av1.h

```diff
@@ -0,0 +1,546 @@
+     *  Valid only when apply_grain equals 1.
+     */
+    VASurfaceID             current_display_picture;
+
+   /** \brief Picture resolution minus 1
+     *  Picture original resolution. The value may not be multiple of 8.
+     *  value range [0..65535]
+     */
+    uint16_t                frame_width_minus1;
+    uint16_t                frame_height_minus1;
+
+   /** \brief Output frame buffer size in unit of tiles
+     *  Valid only when large_scale_tile equals 1.
+     *  value range [0..65535]
+     */
+    uint16_t                output_frame_width_in_tiles_minus_1;
+    uint16_t                output_frame_height_in_tiles_minus_1;
+
+    /** \brief Surface indices of reference frames in DPB.
+     *
+     *  Contains a list of uncompressed frame buffer surface indices as references.
+     *  Application needs to make sure all the entries point to valid frames
+     *  except for the key frame by checking ref_frame_id[]. If missing frame
+     *  is identified, application may choose to perform error recovery by
+     *  pointing problematic index to an alternative frame buffer.
+     *  Driver is not responsible to validate reference frames' id.
+     */
+    VASurfaceID             ref_frame_map[8];
+
+     /** \brief Reference frame indices.
+     *
+     *  Contains a list of indices into ref_frame_map[8].
+     *  It specifies the reference frame correspondence.
+     *  The indices of the array are defined as [LAST_FRAME – LAST_FRAME,
+     *  LAST2_FRAME – LAST_FRAME, …, ALTREF2_FRAME – LAST_FRAME], where each
+     *  symbol is defined as:
+     *  enum{INTRA_FRAME = 0, LAST_FRAME, LAST2_FRAME, LAST3_FRAME, GOLDEN_FRAME,
+     *  BWDREF_FRAME, ALTREF_FRAME};
+     */
+    uint8_t                 ref_frame_idx[7];
+
+    /** \brief primary reference frame index
+     *  Index into ref_frame_idx[], specifying which reference frame contains
+     *  propagated info that should be loaded at the start of the frame.
+     *  When value equals PRIMARY_REF_NONE (7), it indicates there is
+     *  no primary reference frame.
+     *  value range [0..7]
+     */
+    uint8_t                 primary_ref_frame;
+
+    uint8_t                 order_hint;
+
+    VASegmentationStructAV1 seg_info;
+    VAFilmGrainStructAV1    film_grain_info;
+
+    /** \brief tile structure
+     *  When uniform_tile_spacing_flag == 1, the value of tile_cols and
+     *  tile_rows need to be power of 2, and width_in_sbs_minus_1[] and
+     *  height_in_sbs_minus_1[] should be ignored, which will be generated
+     *  by driver based on tile_cols and tile_rows.
+     */
+    uint8_t                 tile_cols;
+    uint8_t                 tile_rows;
+    uint16_t                width_in_sbs_minus_1[63];
+    uint16_t                height_in_sbs_minus_1[63];
+
+    /** \brief number of tiles in large scale tile list
+     *  Valid only when large_scale_tiles == 1.
+     */
+    uint16_t                tile_count_minus_1;

```

Comment:
Why we need this. is it equals to tile_cols * tile_rows  - 1?

Reply:
This is one tile list semantic for large scale specifically, it is equal to the number of tile_list_entry in the list minus 1.

Reply:
ok, thanks for explain.